metalearn/metafeatures/metafeatures_schema.json

```diff
@@ -11,7 +11,7 @@
             "properties": {
                 "value": {
                     "type": [
-                        "number", "string"
+                        "number", "string", "array"
```

Comment:
column-level metafeatures should not be included with dataset-level metafeatures